typescript/sdk/src/router/ProxiedRouterChecker.ts

```diff
@@ -11,7 +11,7 @@ export abstract class ProxiedRouterChecker<
 > extends HyperlaneRouterChecker<Factories, App, Config> {
   async checkOwnership(chain: ChainName): Promise<void> {
     const config = this.configMap[chain];
-    let ownableOverrides = {};
+    let ownableOverrides = config.ownerOverrides;
```

Comment:
something for the impending future: i'm not sure if new modules use/respect ownerOverrides at all, worth adding to the cli reqs sheet

Reply:
great call